src/lib/api.js

```diff
@@ -25,8 +27,17 @@ export default async function API(): Promise<ApolloClient> {
 		headers.Authorization = `Bearer ${ token.raw }`;
 	}
 
+	const unauthorizedLink = onError( ( { networkError } ) => {
+		if ( networkError.statusCode === 401 ) {
+			console.error( chalk.red( 'Error:' ), 'You are unauthorized to perform this request' );
```

Comment:
Discussed on Slack - let's make this message a little more verbose by including something that says "Please log in again" with details on how to logout `vip logout`.